tests/test_registrations/test_retractions.py

```diff
@@ -562,7 +550,6 @@ def test_approval_cancels_active_embargoes_on_descendant_nodes(self, mock_update
         for node in descendants:
             assert_true(node.is_retracted)
 
-        assert mock_update_share.called
```

Comment:
I'm am going try to phase out the method of selectively mocking functions for tests and focus on what actually gets sent in the request to SHARE (or whatever service). It's difficult to mock certain objects, especially client, I ran into that problem a lot on this ticket.

I'll try to add some additional assertions here, but I really want to avoid this level of method monkey mocking for features in the future, especially when responses can be used

Reply:
yeah that makes sense -- was thinking you could use the `responses`-based mock (yielded from a `mock_share` fixture or similar) to assert the correct stuff got sent to SHARE